### ISSUE TITLE
cura 4.0 & 4.1 (beta) - importing profile problem

### ISSUE THREAD

**Issue by MAORUBIN** [Ultimaker/Cura#5764]
Cura 4.0 and Cura 4.1 (Beta)

windows 10 pro

Ultimaker s5

saving a profile then uploading it

shows massage: " failed to import profile from (file location) could not find a quality type.."

**Comment by nallath** [comment]
I've just tried this and it works just fine. Could you also share your log files? There should be some more hints there telling us what went wrong.

**Comment by MAORUBIN** [comment]
I just created a new profile and uploaded it successfully. 

this is the file and the error massage i get from this file, were saved in the same way. 

![1](https://user-images.githubusercontent.com/50827987/58026075-637ecb00-7b1e-11e9-9fac-97a3f96fe696.JPG)
[ABS.zip](https://github.com/Ultimaker/Cura/files/3198169/ABS.zip)

**Comment by Ghostkeeper** [comment]
Yeah, I could reproduce this, both with Maorubin's ABS file but also with a file that I created myself.

The reproduce steps:
1. Switch to some configuration that has no quality profiles for it. In my case, I set it to use a 0.25mm nozzle with a 0.8mm nozzle, but it'll probably also work if you just create a custom material type or whatever.
2. Change a setting, save it to a profile.
3. Export the profile.
4. Import the profile again. This fails.
